integration/spark/src/main/java/openlineage/spark/agent/lifecycle/plan/wrapper/OutputDatasetVisitor.java

```diff
@@ -0,0 +1,36 @@
+package openlineage.spark.agent.lifecycle.plan.wrapper;
+
+import io.openlineage.client.OpenLineage;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan;
+import scala.PartialFunction;
+import scala.runtime.AbstractPartialFunction;
+
+public class OutputDatasetVisitor
```

Comment:
Same comment here.

Reply:
We should have at least OutputDataset cause we need to set OutputStatisticsOutputDatasetFacet to it